src/main/scala/DocGenerator.scala

```diff
@@ -245,7 +251,45 @@ object LatexDocGenerator extends DocGenerator {
                     """
   }
 
+
+  def hypertarget(e: {def id: Seq[Tree]}, text: Option[String]): String = {
+    val t = text.getOrElse("")
+    if (e.id.isEmpty)
+      t
+    else
+      s"\\hypertarget{${link(e.id)}}{$t}"
+  }
+
+  def hyperlink(e: {def id: Seq[Tree]}, text: Option[String]) = {
+    val t = text.getOrElse("")
+    if (e.id.isEmpty)
+      t
+    else
+      s"\\hyperlink{${link(e.id)}}{$t}"
+  }
+
+  def link(tpe: Seq[Tree]) = {
+    val termToTerm = "."
+    val termToType = "."
+    val typeToType = "#"
+    val typeToTerm = "_"
```

Comment:
It sort of feels like it makes sense for docs, even if it doesn't for code:
```scala
trait T {
  /** how many foos
   */
  val fooCount: Int
}
object T extends T {
  /** two foos
   */ 
 val fooCount = 2
}
```

You want docs for both the `fooCount` in the `object` and the `fooCount` in the `trait`... even if there isn't an actual expression of the path to the abstract `fooCount` in the `object` in Scala proper...